test/manipulation.test.js

```diff
@@ -891,6 +894,51 @@ describe('manipulation', function() {
     });
   });
 
+  describe('updateOrCreate when forceId is true', function() {
+    var Post;
+    before(function(done) {
```

Comment:
no lambdas for mocha hooks -- see http://loopback.io/doc/en/contrib/style-guide.html#layout-of-test-files